problems/sevenBoom.js

```diff
@@ -1,11 +1,23 @@
 /** Write a function that returns all the values from 1 to n inclusive, replacing all numbers that are a multiple of seven, or contain seven with "BOOM".
-* Sample Input: 20
-* Sample Output: [1,2,3,4,5,6,'BOOM',8,9,10,11,12,13,'BOOM',15,16,'BOOM',18,19,20]
-* @param {number} n - The number to count up to
-* @returns {number[]} - An array matching the pattern described above
-*/
-function sevenBoom() {
-
+ * Sample Input: 20
+ * Sample Output: [1,2,3,4,5,6,'BOOM',8,9,10,11,12,13,'BOOM',15,16,'BOOM',18,19,20]
+ * @param {number} n - The number to count up to
+ * @returns {number[]} - An array matching the pattern described above
+ */
+function sevenBoom(n) {
+  let arr = [];
+  for (let i = 1; i <= n; i++) {
+    if (i % 10 === 7) {
```

Comment:
This test is giving you a bit of a false positive b/c numbers like 271 should be BOOM but your test won't do that. Revisit and see if you can get this work for all numbers.